docs/file-formats/xml-data-file.md

```diff
@@ -130,22 +130,33 @@ So, in the following example,
 the bitstreams don't inherit the permissions from their resource:
 
 ```xml
+<permissions id="open">
+  <allow group="ProjectAdmin">CR</allow>
+  <allow group="ProjectMember">D</allow>
+  <allow group="KnownUser">V</allow>
+  <allow group="UnknownUser">V</allow>
+</permissions>
+<permissions id="restricted">
+  <allow group="ProjectAdmin">CR</allow>
+  <allow group="ProjectMember">D</allow>
+</permissions>
 <resource ...>
-    <bitstream permissions="prop-default">images/EURUS015a.jpg</bitstream>
+    <bitstream permissions="open">images/EURUS015a.jpg</bitstream>
 </resource>
 <resource ...>
-    <bitstream permissions="prop-restricted">images/EURUS015a.jpg</bitstream>
+    <bitstream permissions="restricted">images/EURUS015a.jpg</bitstream>
 </resource>
 <resource ...>
     <bitstream>images/EURUS015a.jpg</bitstream>
 </resource>
 ```
 
-To take as example `KnownUser`, i.e. a logged-in user who is not member of the project:
+So if you upload this data, and then log in as `KnownUser`, i.e. a logged-in user who is not member of the project, 
+you will see the following:
 
-- With `permissions="prop-default"`, he has `V` rights on the image: Normal view.
-- With `permissions="prop-restricted"`, he has `RV` rights on the image: Blurred image.
-- With a blank `<bitstream>` tag, he has no rights on the image: No view possible. Only users from `ProjectAdmin` 
+- With `permissions="open"`, you have `V` rights on the image: Normal view.
+- With `permissions="restricted"`, you have `RV` rights on the image: Blurred image.
```

Comment:
This is not true isn't it, because if it is restricted you do not see it, if you have restricted-view you have RV...

Reply:
Good catch, thank you.